web_src/js/features/admin/selfcheck.js

```diff
@@ -0,0 +1,31 @@
+import {toggleElem} from '../../utils/dom.js';
+import {POST} from '../../modules/fetch.js';
+
+const {appSubUrl} = window.config;
+
+export async function initAdminSelfCheck() {
+  const elCheckByFrontend = document.querySelector('.page-content.admin .self-check-by-frontend');
```

Comment:
Use an id for performance?

Reply:
Done in 45022537e57227399639e694402a8c98af6e8846